app/components/Views/WalletConnectSessions/index.test.tsx

```diff
@@ -1,13 +1,55 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { waitFor } from '@testing-library/react-native';
 import WalletConnectSessions from './';
+import AsyncStorage from '../../../store/async-storage-wrapper';
+import renderWithProvider from '../../../util/test/renderWithProvider';
 
 describe('WalletConnectSessions', () => {
-  it('should render correctly', () => {
-    const wrapper = shallow(
+  it('does not render when not ready', () => {
+    const { toJSON } = renderWithProvider(
+      <WalletConnectSessions navigation={{ setOptions: () => null }} />,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('renders empty component with no active sessions', async () => {
+    jest.spyOn(AsyncStorage, 'getItem').mockResolvedValue(null);
+
+    const { getByTestId, toJSON } = renderWithProvider(
+      <WalletConnectSessions navigation={{ setOptions: () => null }} />,
+    );
+
+    await waitFor(() => {
+      const emptyMessage = getByTestId('wallet-connect-sessions-screen');
```

Comment:
```suggestion
      const emptyMessage = getByTestId(ExperimentalSelectorsIDs.CONTAINER);
```

You'll have to import ExperimentalSelectorsIDs